README.md

```diff
@@ -1,6 +1,64 @@
-# edgex-endorsement
+# EdgeX Endorsement
 
 ## Introduction
 
-Code, tools, and documentation for validating and endorsing device profiles written by third parties.
+Code, tools, and documentation for validating and endorsing device profiles written by third parties. At a high level you'll need to complete five steps to be eligible for the EdgeX Endorsement Program which will ensure that data provided meets all required components. 
+
+1) Deploy EdgeX using one of the provided cloud templates. Alternatively, you can leverage the latest released compose-file provided here: This ensures a vanilla version of EdgeX is being used without modification to ensure the highest compatibility.
+2)  Compose and deploy your device profile using the Device Profile Modeling tool provided [here](https://www.edgexfoundry.org/tbd). This will ensure the profile is validated and has all required components.
+2) Run your device locally, configured to point at the cloud instance of the deployed EdgeX to send data to either an MQTT or REST device service.
```

Comment:
Technically, this is your third step, not a second!  😄 

This reminds me that we should provide a pre-requisites lists that would list, like, having a device with the ability to send (at least mocked/staged) data.

[Thinking out loud here]  There is also a need for a paragraph that provides an overview/transparency of endorsement process implementation.  A diagram here would be helpful.

[Further thinking out loud here]  I'm wondering if we need to walk the user through this step as there is a lot of hand-waving over some heavy lifts:
1. Deploying the device profile to the appropriate device service
2. Configuring the appropriate device service with the device configuration
3. How to find needed data from one's EdgeX instance that is needed for the device's configuration.

Reply:
Yes I like this, pre reqs would be good -- outline things to be aware of. Yes these are the details I need help with :)